ons-spark/_toc.yml

```diff
@@ -51,6 +51,7 @@ parts:
   chapters:
   - file: spark-analysis/logistic-regression
   - file: spark-analysis/visualisation
+  - file: spark-analysis/Cramer_v
```

Comment:
Shouldn't be like this ? I think when you created this branch bin-con-variables was not implemented 

- caption: Analysis in Spark
 chapters:
  - file: spark-analysis/logistic-regression
  - file: spark-analysis/visualisation
  - file: spark-analysis/bin-continuous-variable
  - file: spark-analysis/Cramer_v

Reply:
Merged main into cramers_v branch to solve conflict. It was bin-con-variables page on _toc